### PR TITLE
Rename protos pkg name, add bft smart config to configtx encoder

### PR DESCRIPTION
**Pull request by C0rWin**
Add ability to encode smart bft configuration from configtx.yaml into
OSN configuration, load tls certs and initialize identities.

Change-Id: I9b94006879894dd8d28b62c515283842638dfc20
Signed-off-by: Artem Barger <bartem@il.ibm.com>

### CODE REVIEW

File: protos/orderer/smartbft/configuration.proto [hunk 1]
```diff
@@ -25,25 +25,14 @@ message Consenter {
     uint32 consenter_id = 1;
     string host = 2;
     uint32 port = 3;
-    bytes identity = 4; // consenter serialized identity
-    bytes client_tls_cert = 5;
-    bytes server_tls_cert = 6;
+    string msp_id = 4;
+    bytes identity = 5; // consenter serialized identity
```
Comment (yacovm) [inline protos/orderer/smartbft/configuration.proto]: this isn't the serialized identity anymore, but the certificate, no?

File: sampleconfig/configtx.yaml [hunk 2]
```diff
@@ -372,6 +372,28 @@ Orderer: &OrdererDefaults
             # SnapshotIntervalSize defines number of bytes per which a snapshot is taken
             SnapshotIntervalSize: 20 MB
 
+    # SmartBFT defines configuration which must be set when the "smartbft"
+    # orderertype is chosen.
+    SmartBFT:
+        # The set of BFT replicas for this network.
```
Comment (yacovm) [inline sampleconfig/configtx.yaml:378]: what about the MSPID ?

**Review by yacovm — COMMENTED** [review]
left some comments

File: protos/orderer/smartbft/configuration.proto (hunk 1 above)
Comment (C0rWin) [reply protos/orderer/smartbft/configuration.proto]: This is serialized identity.

File: sampleconfig/configtx.yaml (hunk 2 above)
Comment (C0rWin) [reply sampleconfig/configtx.yaml:378]: forgot, adding it.

**Pull request merged**